src/app/api/faucet/withdraw/route.ts

```diff
@@ -0,0 +1,59 @@
+import { NextResponse } from 'next/server';
+import { initializeClients, walletClient, publicClient, account } from '../../../../lib/client';
+import { abi, address } from '../../../../lib/constants';
+import { sepolia } from 'viem/chains';
+
+async function withdraw(userAddress: string) {
+    try {
+        initializeClients();
```

Comment:
you can just say {walletClient, account, publicClient} = initializeClients() if you reutrn the three variables from the function